src/main/java/com/example/application/ui/horizontal/community/Objects.java

```diff
@@ -0,0 +1,98 @@
+package com.example.application.ui.horizontal.community;
+
+public class Objects implements Cloneable {
+    private int id;
+    private String cate;
```

Comment:
cate as in category? Please use better names so others can understand better what the variable stands for. Please change the name